### DISCUSSION TITLE
Recommendations on how to handle OpenSSH deployments

### DISCUSSION THREAD

**Discussion by Saxomania** [PowerShell/Win32-OpenSSH#2084] [Q&A] [answered]
Hi,

short introduction:

we manage ~3000 Windows clients, with SCCM and we are using LTSC versions of Windows 10.
We offer to enable OpenSSH Feature in Softwarecenter (done via: "%SystemRoot%\system32\WindowsPowerShell\v1.0\powershell.exe" Add-WindowsCapability -Online -Name OpenSSH.Client~~~~0.0.1.0) and now i see a strange thing:

Windows 10 1809 LTSC (fresh installation) --> OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5
Windows 10 2109 LTSC  (Upgrade from 1809 to 2109) --> OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5 (seen on one device, try reproduce it)
Windows 10 2109 LTSC (fresh installation) --> OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2

Any idea why its an older OpenSSH/LIbreSSL version on the upgraded device in compare to a fresh 2109 installation?

Whats the recommendation to the "built-in" OpenSSH feature? Keep at this version or (as always recommended to use latest version) better remove it and install always the latest MSI version from here? Are there any differences in stability and reliability?

It shouldnt be problem i guess to supersede the built-in feature with the latest from via SCCM/Softwarecenter (remove OpenSSH feature, install OpenSSH MSI). Maybe its also better, i will have a better software inventory because its listed in installed programs.

Thanks Sascha

**Comment by maertendMSFT** [comment]
Is this reproducible on multiple devices?

The difference between in-box Win32-OpenSSH and the betas that are released here are the support avenues.  The in-box OpenSSH is all that is covered by Windows support.  Installations from GitHub are supported via GitHub issues.  There is no difference in the quality of the releases.

**Comment by Saxomania** [comment, marked as answer]
>Is this reproducible on multiple devices?

no not yet, as far as i have seen its only on this device but i had no eyes on this topic so maybe the only maybe not

i will do some tests on testing devices

>The difference between in-box Win32-OpenSSH and the betas that are released here are the support avenues. The in-box OpenSSH is all >that is covered by Windows support. Installations from GitHub are supported via GitHub issues. There is no difference in the quality of >the releases.

ok thanks, so i will think about

**Comment by Saxomania** [comment]
i have now another device with the same problem

Windows 10 2109 LTSC (Upgrade from 1809 to 2109) --> OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5 

did some checks but no clue about it


1. OpenSSH was already enabled before upgrade
2. disabled and enabled feature again, still old version
3. removed the old windows version (via disk clean up), disabled and enabled feature again, still old version
4. reboot after removing, disabled and enabled feature again, still old version

Dont know what to check next or how to workaroung. Maybe should do a new try but upgrade with disabled OpenSSH feature.
